data/data/aws/bootstrap/main.tf

```diff
@@ -120,8 +120,16 @@ resource "aws_instance" "bootstrap" {
   volume_tags = "${var.tags}"
 }
 
-resource "aws_elb_attachment" "bootstrap" {
-  count    = "${var.elbs_length}"
-  elb      = "${var.elbs[count.index]}"
-  instance = "${aws_instance.bootstrap.id}"
+resource "aws_lb_target_group_attachment" "bootstrap_public" {
```

Comment:
This is already namespaced by living in the bootstrap module.  Maybe we can just call this `public`?

```hcl
 resource "aws_lb_target_group_attachment" "public" {
```

Reply:
:+1: